action/protocol/rewarding/reward_test.go

```diff
@@ -382,18 +383,18 @@ func TestProtocol_NoRewardAddr(t *testing.T) {
 		nil,
 		2,
 		2,
-		cfg.Genesis.DardanellesNumSubEpochs,
-		cfg.Genesis.ProductivityThreshold,
-		cfg.Genesis.ProbationEpochPeriod,
-		cfg.Genesis.UnproductiveDelegateMaxCacheSize,
-		cfg.Genesis.ProbationIntensityRate)
+		g.DardanellesNumSubEpochs,
+		g.ProductivityThreshold,
+		g.ProbationEpochPeriod,
+		g.UnproductiveDelegateMaxCacheSize,
+		g.ProbationIntensityRate)
 	require.NoError(t, err)
 	pp, err := poll.NewGovernanceChainCommitteeProtocol(
 		nil,
 		committee,
 		uint64(123456),
 		func(uint64) (time.Time, error) { return time.Now(), nil },
-		cfg.Chain.PollInitialCandidatesInterval,
+		bcf.PollInitialCandidatesInterval,
```

Comment:
directly use `blockchain.DefaultConfig.PollInitialCandidatesInterval` here, no need to define `bcf`